sys-apps/pciutils/pciutils-3.6.0.recipe

```diff
@@ -0,0 +1,140 @@
+SUMMARY="Utilities for inspecting and manipulating PCI devices"
+DESCRIPTION="The PCI Utilities are a collection of programs for inspecting \
+and manipulating configuration of PCI devices, all based on a common \
+portable library libpci which offers access to the PCI configuration space \
+on a variety of operating systems.
+
+The utilities include:
+
+- lspci - displays detailed information about all PCI buses and devices in \
+the system
+- setpci - allows reading from and writing to PCI device configuration \
+registers. For example, you can adjust the latency timers with it."
+HOMEPAGE="https://mj.ucw.cz/sw/pciutils/"
+COPYRIGHT="1997-2018 Martin Mares"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="ftp://atrey.karlin.mff.cuni.cz/pub/linux/pci/pciutils-$portVersion.tar.gz"
```

Comment:
The [kernel.org](https://www.kernel.org/pub/software/utils/pciutils/) mirror should also be added, to aid people behind firewalls.